scripts/provision_layers.py

```diff
@@ -0,0 +1,73 @@
+from argparse import ArgumentParser
+import logging
+from pathlib import Path
+import subprocess
+import sys
+from zipfile import ZipFile
+
+import boto3
+
+from azul import config
+from azul.logging import configure_script_logging
+
+log = logging.getLogger(__name__)
+
+
+class LayersProvisioner:
```

Comment:
This merely stages the layer package and TerraForm actually provisions the lambda layer, correct? If so, could the name of the class and the script reflect that?

Reply:
Correct. This class name is a holdover from my original implementation.